models/org.go

```diff
@@ -495,7 +499,7 @@ func (org *User) GetUserRepositories(userID int64, page, pageSize int) ([]*Repos
 	repos := make([]*Repository, 0, pageSize)
 	// FIXME: use XORM chain operations instead of raw SQL.
 	if err = x.Sql(fmt.Sprintf(`SELECT repository.* FROM repository
```

Comment:
Probably not, but this code is not part of the pull request. It could be a part of another one (eg : rewrite and optimize all SQL queries)